commons-test/src/main/java/org/infinispan/commons/test/annotation/TestForIssue.java

```diff
@@ -0,0 +1,25 @@
+package org.infinispan.commons.test.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * A documentation annotation for notating what JIRA issue is being tested.
+ * <p>
+ * Copied from <a href="https://github.com/hibernate/hibernate-orm">Hibernate ORM project</a>.
+ *
+ * @author Steve Ebersole
+ */
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Does it really need to be `RUNTIME` ?

Reply:
I didn't ask me that question. Changed to `SOURCE` it seems to be fine.